### ISSUE TITLE
Windows 10 - Source Path Too Long

### ISSUE THREAD

**Issue by tugrul** [Glavin001/atom-beautify#1045]
### Description

I tried update the packages but there is no respond from Atom. Also I uninstalled Atom, tried to remove folder `C:\Users\abcdefg\.atom\`

```
C:\Users\abcdefg\.atom\packages\atom-beautify\node_modules\analytics-node\node_modules\superagent-proxy\node_modules\proxy-agent\node_modules\pac-proxy-agent\node_modules\pac-resolver\node_modules\regenerator\node_modules\defs\node_modules\yargs
```

### Steps to Reproduce

1. Try to update atom-beautify module
2. Uninstall Atom
3. Try to remove folder `C:\Users\abcdefg\.atom\`

**Comment by prettydiff** [comment]
This defect is due to Atom using an old version of Node/NPM.  This is resolved with NPM 3 which intelligently collapses and normalizes the dependency tree.

**Comment by LuisaMarvin** [comment]
Long path error is a very common problem in Windows users face.You can try using Long Path Tool if your computer is giving errors like, “Path too long”, “Path too long – aborting (error code 80/1)”, 0x80010135 Path too long”, “Error cannot delete file: cannot read from source file or disk” or “File cannot be copied”, you can use Long Path Tool to resolve the problem. 

Long Path Tool is the perfect utility that can solve all these issues. Get support of this product by subscribing it online.

**Comment by Samantha258** [comment]
“Long Path Tool” is very helpful for this error !
You can use to solve this problem

**Comment by WrightSanches** [comment]
If I can still remember we have experience the same problem before, but in a windows server then one of my colleagues has found a solution with the help of a program called: “Long path tool”. His not working with us any more so I can't be sure if that's really the program. Hope that helps.

**Comment by Glavin001** [comment]
This should always be solved by using Atom >= 1.10.x.
See https://github.com/atom/atom/releases/tag/v1.10.0

Related articles:
- https://discuss.atom.io/t/long-file-name-path-issue-on-windows-with-atom-packages/30552
- https://github.com/atom/atom/issues/5109

**Comment by SergiyMike** [comment]
You can try "Long Path Tool" as it deletes, copies and renames files/folders those have long paths. I personally found all of these functions working with great success. :)
